apps/wallet-mobile/src/features/WalletManager/SelectWalletFromList/SelectWalletFromListScreen.tsx

```diff
@@ -119,6 +120,8 @@ export const SelectWalletFromList = () => {
         <Space height="m" />
 
         <OnlyDevButton />
+
+        <Space height="l" />
```

Comment:
REAL FIX HERE @stackchain @michaeljscript  @jorbuedo

Reply:
the rest is just moving folders and files

Reply:
I think it was already moved in dev, maybe rebase and apply only this update

Reply:
its a new refactor. Wallet list -> to WalletManager folder

not changed in develop yet https://github.com/Emurgo/yoroi/tree/develop/apps/wallet-mobile/src/features/SetupWallet/useCases/SelectWalletFromList